### ISSUE TITLE
Missing header `<cstdint>` 

### ISSUE THREAD

**Issue by correaa** [QMCPACK/qmcpack#4792]
**Describe the bug**

Missing header `<cstdint>` in `qmcpack/src/Platforms/OMPTarget/ompBLAS.cpp`  for compilation with GCC 13.2

https://gitlab.com/correaa/boost-multi/-/jobs/5346169350#L1644

**To Reproduce**

With debian:testing and GCC 13.2:

```bash
$ cmake -DCMAKE_C_COMPILER=mpicc -DCMAKE_CXX_COMPILER=mpicxx -DBUILD_AFQMC=1 -DQMC_MIXED_PRECISION=1 -DCMAKE_BUILD_TYPE=Debug -DMPIEXEC_PREFLAGS="--allow-run-as-root;--bind-to;none" ..
$ make
```

**Expected behavior**

Compilation should work

**System:**

Docker image with debian:testing

**Comment by ye-luo** [comment]
Do you have access to this docker image? Could you make a fix?

**Comment by correaa** [comment]
I can me a a fix in the code itself, `/ompBLAS.cpp`, would that be ok?

**Comment by ye-luo** [comment]
> I can me a a fix in the code itself, `/ompBLAS.cpp`, would that be ok?

Preferably you test your fix under that image just in case of other similar issues.

**Comment by ye-luo** [comment]
@correaa your PR was open in your own fork repo no the upstream qmcpack.

**Comment by correaa** [comment]
I am confused, I declared QMCPACK as upstream. Is there an easy way to transfer the PR? 

```
git remote add upstream git@github.com:QMCPACK/qmcpack.git
git fetch upstream
git reset --hard upstream/develop
git push
```

**Comment by ye-luo** [comment]
Your [PR](https://github.com/correaa/qmcpack/pull/5) is not in qmcpack repo https://github.com/QMCPACK/qmcpack/pulls

**Comment by ye-luo** [comment]
Fixed by #4796
